bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/MiIoQuantiyTypes.java

```diff
@@ -33,31 +36,49 @@
 @NonNullByDefault
 public enum MiIoQuantiyTypes {
 
-    CELCIUS(SIUnits.CELSIUS),
-    FAHRENHEIT(ImperialUnits.FAHRENHEIT),
-    SECOND(SmartHomeUnits.SECOND),
-    MINUTE(SmartHomeUnits.MINUTE),
-    HOUR(SmartHomeUnits.HOUR),
-    SECONDS(SmartHomeUnits.SECOND),
-    MINUTES(SmartHomeUnits.MINUTE),
-    HOURS(SmartHomeUnits.HOUR),
-    AMPERE(SmartHomeUnits.AMPERE),
-    WATT(SmartHomeUnits.WATT);
+    CELCIUS(SIUnits.CELSIUS, "C"),
+    FAHRENHEIT(ImperialUnits.FAHRENHEIT, ""),
+    SECOND(SmartHomeUnits.SECOND, "seconds"),
+    MINUTE(SmartHomeUnits.MINUTE, "minutes"),
+    HOUR(SmartHomeUnits.HOUR, "hours"),
+    AMPERE(SmartHomeUnits.AMPERE, ""),
+    WATT(SmartHomeUnits.WATT, ""),
```

Comment:
You can remove the empty strings then. Do so for the others as well.
```suggestion
    AMPERE(SmartHomeUnits.AMPERE),
    WATT(SmartHomeUnits.WATT),
```